concourse/pipelines/guest-package-build.yaml

```diff
@@ -285,6 +293,21 @@ jobs:
           universe: cloud-yum
           repo: google-guest-agent-el8
           environment: stable
+  - task: publish-time-since-stable
+    file: guest-test-infra/concourse/tasks/publish-time-since.yaml
+    vars:
+      last_publish_date: ((.:last-stable-date))
+      package_name: "google-guest-agent"
+  - put: guest-agent-tag
+    params:
+      name: last-stable-tag/stable  # file contains only the word 'stable'
+      tag: last-stable-tag/stable
+      commitish: guest-agent-tag/commit_sha
+  - put: guest-agent-tag
+    params:
+      name: last-stable-tag/stable-today  # file contains 'stable-YYYYMMDD'
+      tag: last-stable-tag/stable-today
+      commitish: guest-agent-tag/commit_sha
```

Comment:
Does this produce a 'stable-today' tag on the GH 'releases' page? Or does it produce 'stable-YYYYMMDD'?

Reply:
the latter; 'stable-today' is the name of the file within the Concourse job